src/Umbraco.Core/Services/IdkMap.cs

```diff
@@ -44,12 +45,12 @@ public Attempt<int> GetIdForKey(Guid key, UmbracoObjectTypes umbracoObjectType)
             }
 
             if (val == null) return Attempt<int>.Fail();
-            id = val.Value;
```

Comment:
I'll be super-annoying: can we just `new TypeId<int>(...)` when `_key2Id[key] = ...` and therefore change even less lines? Same for the `GetKeyForId` method.